src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+
+public interface OperationHandler {
+    Storage storage = new Storage();
+    void update(String key, Integer value);
```

Comment:
Bad name. Interface have name OperationHandler it is obvious that it should HANDLE something not update